runtime/firecracker-runc-config.json.example

```diff
@@ -0,0 +1,155 @@
+{
+    "ociVersion": "1.0.1-dev",
+    "process": {
+        "terminal": false,
+        "user": {
+            "uid": 0,
+            "gid": 0
+        },
+        "env": [
+            "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
```

Comment:
nit: does firecracker actually need a PATH to be set? If so, can we set it more specifically than this?